examples/platform/infineon/cyw30739/matter_config.cpp

```diff
@@ -237,6 +238,9 @@ void CYW30739MatterConfig::InitApp(void)
 #endif
 
     // Init Matter Server
+    const Clusters::GeneralDiagnosticsEnabledAttributes enabledAttributes = Clusters::GeneralDiagnosticsEnabledAttributes();
+    static Clusters::GeneralDiagnosticsCluster<Clusters::DeviceLayerGeneralDiagnosticsLogic> cluster(enabledAttributes);
```

Comment:
CYW30739 does not use CodegenIntegration? This seems odd ... some comments explaing would be good